src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasureRecord.test.ts

```diff
@@ -0,0 +1,76 @@
+import { calculatedMeasures } from "../__test_resources__/uiCalculated_measuresFolder";
+import { migrateCalculatedMeasureRecord } from "./migrateCalculatedMeasureRecord";
+
+const distinctCountCity = calculatedMeasures.children.content.children["196"];
+const expGammaSum = calculatedMeasures.children.content.children["501"];
+
+describe("migrateCalculatedMeasureRecord", () => {
+  it("transforms the serialized definition of a calculated measure created with ActiveUI 5.0 which contains a `FORMAT_STRING` property, into one that is natively supported by ActivePivot", () => {
+    const migratedRecord = migrateCalculatedMeasureRecord(
+      distinctCountCity,
+      "Distinct count city",
+    );
+    expect(migratedRecord).toMatchInlineSnapshot(`
+      Object {
+        "entry": Object {
+          "canRead": true,
+          "canWrite": true,
+          "content": "{\\"className\\":\\"com.quartetfs.biz.pivot.definitions.impl.CalculatedMemberDescription\\",\\"additionalProperties\\":{},\\"uniqueName\\":\\"[Measures].[Distinct count city]\\",\\"expression\\":\\"Count(Descendants([Geography].[City].CurrentMember, [Geography].[City].[City]), EXCLUDEEMPTY)\\",\\"formatStringExpression\\":\\"FORMAT_STRING = \\\\\\"#,###.##\\\\\\"\\"}",
+          "isDirectory": false,
+          "lastEditor": "admin",
+          "owners": Array [
+            "admin",
+          ],
+          "readers": Array [
+            "admin",
+          ],
+          "timestamp": 1666091498549,
+        },
+      }
+    `);
+
+    expect(JSON.parse(migratedRecord.entry.content)).toStrictEqual({
+      additionalProperties: {},
+      className:
+        "com.quartetfs.biz.pivot.definitions.impl.CalculatedMemberDescription",
+      expression:
+        "Count(Descendants([Geography].[City].CurrentMember, [Geography].[City].[City]), EXCLUDEEMPTY)",
+      formatStringExpression: `FORMAT_STRING = "#,###.##"`,
```

Comment:
This is not the expected format.
See this example from the ranch:
![image](https://user-images.githubusercontent.com/29061952/207328691-f2840d1e-a229-48d5-ab4e-6e48344bbf82.png)